src/joystick/vita/SDL_sysjoystick.c

```diff
@@ -86,26 +86,9 @@ static int calc_bezier_y(float t)
     return dest.y;
 }
 
-/* Function to scan the system for joysticks.
- * Joystick 0 should be the system default joystick.
- * It should return number of joysticks, or -1 on an unrecoverable fatal error.
- */
-int SDL_SYS_JoystickInit(void)
+static void
```

Comment:
This whole file has been refactored because the new internal API for joysticks is based on drivers. I just moved a bit of code around to set this driver up, nothing else.